src/components/users/signup/ko/runtime/signup.ts

```diff
@@ -109,6 +109,12 @@ export class Signup {
                     } else {
                         this.usersService.navigateToHome();
                     }
+                } else {
+                    const redirectUrl = this.delegationUrl();
+
+                    if (redirectUrl) {
+                        location.assign(redirectUrl);
```

Comment:
I think we should user a `Router` instead, because it can be mocked for unit testing, have custom logic for redirects/routing, etc.